### PR TITLE
docs: Add .env instructions to the Getting Started page

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
Add environment variable instructions to the Getting Started page. 

Update the order of directions - Move section `API Key` to be in front of `Configure the OnchainKitProvider`

Update all cases of `API Key` to include the `Public` prefix. 

Update all cases of `Key` to be consistent, capital `K` and lowercase `ey`
**Notes to reviewers**

**How has it been tested?**

Screenshot 2024-07-10 at 3 24 27 PM

### CODE REVIEW

File: site/docs/pages/getting-started.mdx [hunk 1]
```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
 
 :::
 
+### API Key
```
Comment (Zizzamia) [inline site/docs/pages/getting-started.mdx:33]: Let's say `Public API Key`, as it is a public one

File: site/docs/pages/getting-started.mdx [hunk 2]
```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
...
+
+#### Obtain an API key from the [Coinbase Developer Platform APIs](https://portal.cdp.coinbase.com/products/onchainkit).
```
Comment (Zizzamia) [inline site/docs/pages/getting-started.mdx:35]: Public API Key

File: site/docs/pages/getting-started.mdx [hunk 3]
```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
...
+/>
+
+#### Create a .env file
+
+Create a new file in your project’s root directory and name it “.env”.
+
+<img
+  alt="OnchainKit copy API KEY"
+  src="https://onchainkit.xyz/assets/getting-started-create-env-file.png"
+  width="auto"
```
Comment (Zizzamia) [inline site/docs/pages/getting-started.mdx:50]: the width is too large, also it feels very odd to see it like that.

File: site/docs/pages/getting-started.mdx [hunk 4]
```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
...
+/>
+
+#### Define your API KEY
```
Comment (Zizzamia) [inline site/docs/pages/getting-started.mdx:53]: Define and Assign should be one step. I understand we are making stupid easy to understand how to do it. But I think we want to speak like midle school level, not elementary school level.

File: site/docs/pages/getting-started.mdx [hunk 5]
```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
...
+```
+
+#### Access the API KEY in your code
+
+```tsx [OnchainProviders.tsx]
+    <OnchainKitProvider 
+      apiKey={process.env.ONCHAINKIT_API_KEY} // [!code focus] 
+      chain={base}
+    >
+      <YourKit />
```
Comment (Zizzamia) [inline site/docs/pages/getting-started.mdx:71]: Let's say `<YourComponents />`

File: site/docs/pages/getting-started.mdx (hunk 3 above)
Comment (cpcramer) [reply site/docs/pages/getting-started.mdx]: Updated! - See image in PR description

File: site/docs/pages/getting-started.mdx (hunk 4 above)
Comment (cpcramer) [reply site/docs/pages/getting-started.mdx]: Combined the stepss:

```
Define your API Key
Set your API key by assigning the value you copied from the Coinbase Developer Platform to the ONCHAINKIT_API_KEY variable.
```

File: site/docs/pages/getting-started.mdx (hunk 1 above)
Comment (cpcramer) [reply site/docs/pages/getting-started.mdx]: Updated all instanced to include the public prefix

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**